src/components/headMetaTags.js

```diff
@@ -44,10 +47,16 @@ class HeadMetaTags extends Component {
     return (
       <Helmet
         meta={[
-          {property: 'og:title', content: {title}}
+          {property: 'og:title', content: {title}},
+          {property: 'og:image', content: {logo}},
+          {property: 'og:description', content: {description}},
+          {property: 'og:url', content: {url}}
         ]}
         script={scripts}
         title={title}
+        logo={logo}
+        description={description}
+        url={url}
```

Comment:
I don't believe these are valid properties on the `Helmet` component:

https://github.com/nfl/react-helmet/blob/release/5.0.0/src/Helmet.js#L30